src/style.css

```diff
@@ -1,3 +1,63 @@
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  height: 60px;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  position: relative;
+  padding: 0 50px;
+}
+
+.image-link {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  line-height: 60px;
```

Comment:
redundant - you centered text via `flex`, no need to set such line height (even if it's on the mockup), but not critical - just FYI